test/e2e/manifest/linux.json

```diff
@@ -6,8 +6,9 @@
             "orchestratorType": "Kubernetes",
             "orchestratorRelease": "1.16",
             "kubernetesConfig": {
+                "useManagedIdentity": true,
+                "userAssignedID": "azurefile-csi-driver-e2e-test-id",
```

Comment:
Did you configure `azurefile-csi-driver-e2e-test-id` identity somewhere?

Reply:
It will create a new identity named `azurefile-csi-driver-e2e-test-id` every time. Currently, aks-engine does not support the use of existing identity during cluster creation so we are relying on `kubetest` to assign "Owner" permission to the identity before running the test.